config/routes.rb

```diff
@@ -1,8 +1,5 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
-
   root to: 'home#index'
-  
   get 'index', to: 'home#index'
-
+  resources :cards
```

Comment:
are you use all routes in this resources? If not, write only useful routes here. 
resources :cards, only: :index